### PR TITLE
Update column names of reference files

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
Some breaking changes introduced recently require updates in our reference file format:

* naming in headers has changed (https://github.com/precice/precice/pull/620)
* ~~numerical differences due to change in Eigen version (https://github.com/precice/systemtests/pull/163): Previously, the Eigen version (which one?) provided via `apt` for Ubuntu16.04 was sufficient. Now we have to build a different Eigen version (which one?) from source. This change introduces [small numerical differences](https://travis-ci.org/precice/systemtests/jobs/645031934) between output and reference **dependendcy on Eigen still needs to be confirmed and implemented**~~ *edit:* dealing with this in #171 and #172.

# Todo

- [x] update column names in [fe-fe](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_fe-fe.Ubuntu1804.home)
- [x] update column names in [nutils-of](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_nutils-of.Ubuntu1804)
- [x] update column names in [su2-ccx](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_su2-ccx)

Note: No updates needed for

* [dealii-of](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_dealii-of)
* [of-ccx](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_of-ccx)
* [of-ccx_fsi](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_of-ccx_fsi.Ubuntu1604.home.PETSc)
* [of-of](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_of-of)
* [of-of_np](https://github.com/precice/systemtests/tree/develop/tests/TestCompose_of-of)
* test_bindings and test_bindings.Ubuntu1804

since log files generated by preCICE are not being compared.

### CODE REVIEW

**Comment by MakisH** [comment]
If we are not sure for the second part, we can just split this into two PR. The first one, we definitely need to add. Then we can investigate the second one separately.

**Comment by BenjaminRodenberg** [comment]
> 
> 
> If we are not sure for the second part, we can just split this into two PR. The first one, we definitely need to add. Then we can investigate the second one separately.

Good point. I will change the title of this PR and check the second issue tomorrow.

File: tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log [hunk 1]
```diff
@@ -7,7 +7,7 @@ Timestep  Iteration  resNorm(0)
 1  6  0.0000014667397339  
 2  1  0.0008861429935086  
 2  2  0.0000009250774064  
-3  1  0.0005705612743593  
+3  100000  0.0005705612743593
```
Comment (MakisH) [inline tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log]: Why this?

Comment (BenjaminRodenberg) [reply tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log]: This is just for the sake of provoking failure (and make really sure that #169 is really an issue)

Comment (BenjaminRodenberg) [reply tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log]: I will of course revert 90ff76e before merging this PR (let's keep this conversation unsresolved as a reminder until I do so)

Comment (BenjaminRodenberg) [reply tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log]: Reverted. See 149a5a9.

**Comment by BenjaminRodenberg** [comment]
Interesting:

* [test 1582.20](https://travis-ci.org/precice/systemtests/jobs/645122303?utm_medium=notification&utm_source=github_status) successful, but column names are wrong 
https://github.com/precice/systemtests/blob/10cfcc4cec0148a8a27d12426bf477bd8efce16e/tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log#L1
* [test 1583.20](https://travis-ci.org/precice/systemtests/jobs/645188339?utm_medium=notification&utm_source=github_status) fails, wrong column names are detected, but there is also a second error
https://github.com/precice/systemtests/blob/90ff76eeebe8ea5cbf4acd405b5175d5d730fd90/tests/TestCompose_nutils-of.Ubuntu1804/referenceOutput/Nutils/precice-Nutils-convergence.log#L10

Am I overseeing something or is this really strange?

PS: I'm already excited about the results of [test 1584.20](https://travis-ci.org/precice/systemtests/jobs/645200413?utm_medium=notification&utm_source=github_status), where the reason for failure of 1583.20 has been reverted.

**Comment by uekerman** [comment]
Needs further updates after https://github.com/precice/precice/pull/656

**Review by BenjaminRodenberg — COMMENTED** [review]
All column names look good now.

**Pull request merged**